components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -65,41 +65,76 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
+                                id={t('announcement_bar.warn_metric_status_ack.text')}
+                                defaultMessage={'Thank you for contacting Mattermost. We will follow up with you soon.'}
+                            />
+                        </React.Fragment>
+                    );
+                }
+
                 type = AnnouncementBarTypes.ADVISOR_ACK;
                 showModal = false;
                 dismissFunc = this.dismissNumberOfActiveUsersWarnMetricAck;
                 isDismissed = this.props.dismissedNumberOfActiveUsersWarnMetricStatusAck;
                 canCloseBar = true;
             } else {
-                message = (
-                    <React.Fragment>
-                        <img
-                            className='advisor-icon'
-                            src={alertIcon}
-                        />
-                        <FormattedMarkdownMessage
-                            id={t('announcement_bar.error.number_active_users_warn_metric_status.text')}
-                            defaultMessage={'You now have over {limit} users. We strongly recommend using advanced features for large-scale servers.'}
-                            values={{
-                                limit: warnMetricStatus.limit,
-                            }}
-                        />
-                    </React.Fragment>
-                );
+                if (warnMetricStatus.id === WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_500) {
+                    message = (
+                        <React.Fragment>
+                            <img
+                                className='advisor-icon'
+                                src={alertIcon}
+                            />
+                            <FormattedMarkdownMessage
+                                id={t('announcement_bar.error.number_active_users_warn_metric_status.text')}
+                                defaultMessage={'You now have over {limit} users. We strongly recommend using advanced features for large-scale servers.'}
+                                values={{
+                                    limit: warnMetricStatus.limit,
+                                }}
+                            />
+                        </React.Fragment>
+                    );
+                } else if (warnMetricStatus.id === WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_POSTS_500K) {
+                    message = (
+                        <React.Fragment>
+                            <img
+                                className='advisor-icon'
+                                src={alertIcon}
+                            />
+                            <FormattedMarkdownMessage
+                                id={t('announcement_bar.number_of_posts_warn_metric_status.text')}
+                                defaultMessage={'You now have over {limit} posts. We strongly recommend using advanced features for large-scale servers.'}
```

Comment:
```suggestion
                                defaultMessage='You now have over {limit} posts. We strongly recommend using advanced features for large-scale servers.'
```

Reply:
Same here.

Reply:
I'll defer to @esadur and @jasonblais on this, they review the texts.

Reply:
These are the banners, and there's only so much space for text. We go into more detail in the modals.